src/App.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import "./App.css";
+
+function App() {
+  let image;
```

Comment:
I believe on this scenario you don't need to initialise the response empty variables.
Maybe the image might be necessary. At least on the flow I used on mine. Converting to FormData. A bit more on that on the next comment.